test/e2e/testdata/fakedata/mapping/product_team_a/product_x/x_mapping.yaml

```diff
@@ -2,7 +2,7 @@ resource:
   name: '//storage.googleapis.com/pmap-dev-manual-bucket'
   provider: 'gcp'
 annotations:
-  traceID: 'fake-bets-pmap-dev-manual-test'
+  traceID: 'fake-bets-pmap-dev-manual-test-mapping'
```

Comment:
Can you also get rid of `bets` here?

Reply:
Done, I also removed `bets` from the workflow name, and yml file name.